src/test/java/Homework19.java

```diff
@@ -0,0 +1,41 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+import java.lang.reflect.Method;
+import java.time.Duration;
+
+
+public class Homework19 extends BaseTest{
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+
+
+        String playlistDeletedMessage ="Deleted playlist";
+
+
+
+
+        provideEmail("bhakti.damle@testpro.io");
+        providePassword("Mangodesk234!");
+        clickSubmit();
+        clickMyPlaylist();
+        clickDeleteMyPlaylist();
+        Assert.assertTrue(getDeletedPlaylistNotification().contains(playlistDeletedMessage));
+
+        }
+
+        }
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._